app.js

```diff
@@ -2,17 +2,20 @@ const express = require('express');
 const app = express();
 const {
     getTopic,
-    getArticle
+    getArticle,
+    getArticleById
 } = require('./controllers')
 const {
     notFound,
-} = require('./controllers.err')
+    handleCustomError
+} = require('./controllers.err');
 
-app.use(express.json());
+app.get('/api/topic', getTopic);
+app.get('/api/articles', getArticle);
+app.get('/api/articles/:article_id', getArticleById);
 
-app.get('/api/topic', getTopic)
-app.get('/api/articles', getArticle)
+app.all('/*', notFound);
 
-app.all('/*', notFound)
+app.use(handleCustomError);

```

Comment:
Thanks for splitting the error handling out, definitely have a app.use(500Errors) at the very bottom of your app.js, this is your catch-all bucket for any errors that don't get into any earlier error handling middleware.